### PR TITLE
sdk: Fake driver

### PR DESCRIPTION
**Pull request by lpabon**
**What this PR does / why we need it**:
The fake driver will be used to create a simple application for
SDK developers to test their APIs.

**Which issue(s) this PR fixes** (optional)
Part of #407

**Special notes for your reviewer**:
With this driver, we can create a simple program for developers to test their SDK APIs

### CODE REVIEW

File: volume/drivers/fake/fake.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+// These functions below implement the volume driver interface.
+//
+
+func (d *driver) Create(
+	locator *api.VolumeLocator,
+	source *api.Source,
+	spec *api.VolumeSpec) (string, error) {
+
+	if spec.Size == 0 {
+		return "", fmt.Errorf("Volume size cannot be zero: buse")
```
Comment (pault84) [inline volume/drivers/fake/fake.go]: why buse?

File: volume/drivers/fake/fake.go [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+	"github.com/sirupsen/logrus"
+
+	"strings"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/volume"
+	"github.com/libopenstorage/openstorage/volume/drivers/common"
+	"github.com/pborman/uuid"
+	"github.com/portworx/kvdb"
```
Comment (pault84) [inline volume/drivers/fake/fake.go]: kvdb code implemented in osd?

File: volume/drivers/fake/fake.go (hunk 1 above)
Comment (lpabon) [reply volume/drivers/fake/fake.go]: Oops, I'll fix that.

File: volume/drivers/fake/fake.go (hunk 2 above)
Comment (lpabon) [reply volume/drivers/fake/fake.go]: Not sure what you mean. It is just imported.

**Pull request merged**